ha/setup/ha_setup.py

```diff
@@ -381,17 +403,20 @@ class UpgradeCmd(Cmd):
     """
     name = "upgrade"
 
-    def __init__(self, args):
+    def __init__(self, args=None):
         """
         Init method.
         """
         super().__init__(args)
+        # self._config = ConfigCmd(args)
+        machine_id = self.get_machine_id()
+        self.s3_instance = Cmd.get_s3_instance(machine_id)
```

Comment:
We cannot use confstore during run-time.
cc @ajaysrivas

Reply:
I think we already discussed this.